src/main/java/hudson/plugins/view/dashboard/test/TestTrendChart.java

```diff
@@ -149,18 +149,18 @@ public int compare(LocalDate d1, LocalDate d2) {
 
          if (run != null) { // execute only if job has builds
         	 LocalDate runDay = new LocalDate(
-        			 run.getTimeInMillis() - dateShift*60000, GregorianChronology.getInstanceUTC());
+        			 run.getTimeInMillis() - dateShift*60000l, GregorianChronology.getInstanceUTC());

```

Comment:
Should it not be `L` rather than `l` as I thought for a second that the change was `6000` to `60001`